<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PB_ActionModifyPositions2DOptions</class>
 <widget class="QWidget" name="PB_ActionModifyPositions2DOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wd_toolbar" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pb_freeMove">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/icons/cursor.png</normaloff>:/icons/cursor.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_mode</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_move">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/icons/move.png</normaloff>:/icons/move.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_mode</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_add">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_mode</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_remove">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/icons/moins.png</normaloff>:/icons/moins.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_mode</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../resource.qrc">:/icons/level.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsb_zval">
              <property name="suffix">
               <string> m</string>
              </property>
              <property name="minimum">
               <double>-9999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>9999.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_1m">
              <property name="text">
               <string>1 m</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_step</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_10cm">
              <property name="text">
               <string>10 cm</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_step</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_1cm">
              <property name="text">
               <string>1 cm</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_step</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_drawPlane">
              <property name="text">
               <string>Dessiner Plan</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_step"/>
  <buttongroup name="buttonGroup_mode"/>
 </buttongroups>
</ui>
